apps/web/src/utils/useOptimisticUserInfo.ts

```diff
@@ -0,0 +1,72 @@
+import { useMemo } from 'react';
```

Comment:
all components and files related to `getOptimisticUserInfo` refactoring was handled in a separate PR, which contains all the screenshots: #1886